frontend/hooks/useStakingContracts.ts

```diff
@@ -7,14 +7,11 @@ import { useServices, useStakingProgram } from '@/hooks';
 export const useStakingContracts = () => {
   const { selectedAgentConfig, selectedAgentType } = useServices();
   const { evmHomeChainId } = selectedAgentConfig;
-  const {
-    activeStakingProgramId,
-    isActiveStakingProgramLoaded,
-    defaultStakingProgramId,
-  } = useStakingProgram();
+  const { isActiveStakingProgramLoaded, selectedStakingProgramId } =
+    useStakingProgram();
 
   const currentStakingProgramId = isActiveStakingProgramLoaded
-    ? activeStakingProgramId || defaultStakingProgramId
+    ? selectedStakingProgramId
```

Comment:
for the ordered list we should get what's in the service but not just active program on-chain (otherwise we show different things in different places)